CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_shopify v0.14.0
+
+[PR #92](https://github.com/fivetran/dbt_shopify/pull/92) includes the following updates:
+## Under the Hood
```

Comment:
```suggestion
## Breaking Changes
```
Same here since this could result in schema changes.

Reply:
updated!